charts/mirrorbits-lite/Chart.yaml

```diff
@@ -1,4 +1,8 @@
 apiVersion: v1
 description: Mirrobits lite helm chart for Kubernetes
 name: mirrorbits-lite
-version: 0.0.2
+version: 0.1.0
+appVersion: v0.5.2
```

Comment:
```suggestion
```
I still think there is zero added value with this (nit)

Reply:
It allows readers to immediately understand that the image tag version scheme is different than the application.
It's recommended by Helm doc but optional and it does not change the behavior (and we have it on the former `mirrorbits` chart?